cluster/charts/rook-ceph/templates/deployment.yaml

```diff
@@ -101,6 +101,12 @@ spec:
         - name: ROOK_CSI_ENABLE_CEPHFS
           value: {{ .Values.csi.enableCephfsDriver | quote }}
 {{- end }}
+{{- if .Values.csi }}
```

Comment:
Could you put one statement for `if .Values.csi` around all the csi settings, instead of repeating it for every setting? That's not directly related to this PR, but could be separate.

Reply:
done